docs/testing.md

```
@@ -0,0 +1,221 @@
+
+   # 期待画像との比較（差分があると *_compare.png が生成される）
+   ./gradlew compareRoborazziDebug
+   ```
+
+画像のファイル名は `{Preview関数名}_{グループ名}.png` になる（`ShowkaseParameterizedTest` 参照）。
+
+### 期待画像の更新・アップロードの仕組み
+
+期待画像は Git 管理ではなく **GitHub Actions のアーティファクトで受け渡している**。手動更新は不要で、main へのマージが実質的な「期待画像の更新」になる。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)